EIPS/eip-4973.md

```diff
@@ -187,6 +187,26 @@ You can find an implementation of this standard in [../assets/eip-4973](../asset
 
 There are no security considerations related directly to the implementation of this standard.
 
+## Privacy Considerations
```

Comment:
@SamWilsn requesting a manual merge. I think authors are free to choose to include additional sections

Reply:
They are not.

Reply:
We used to be allowed to include Appendices, and in my opinion still should, but I don't care to fight that battle again.